src/Pos/Explorer/Socket/Methods.hs

```diff
@@ -159,6 +159,7 @@ subscribeAddr caddr sessId = do
         sessId
         (sformat ("address "%shown) addr)
         (csAddressSubscribers . at addr . non S.empty . at sessId)
+    logDebug $ sformat ("Client #"%shown%" subscribeAddr") sessId
```

Comment:
@Martoon-00 Oh, I think I get it now. At the end of the functions you call `subscribe`/`unsubscribe` and they log everything. So there is no need to add additional logging. Is that correct?